### ISSUE TITLE
OneParExpNormalPrior - skel_fun_inv produce NA

### ISSUE THREAD

**Issue by 0liver0815** [Roche/crmPack#367]
The following code is also part of the JSS publication (Bove et al. 2019) which provides code for the one-parameter CRM (O’Quigley et al. 1990):

https://github.com/Roche/crmPack/blob/d7cb6def0d43ba7cffbb439b367feda027934991/R/Model-class.R#L2621-L2622

Two interpolation functions are introduced to be able to get any DLT prob over the dose range and any dose for a given DLT probability. 

In the function skel_fun_inv the parameter rule=1 is specified. That means that NAs are returned when the prob is outside the range of skel_probs. The Problem is that crmPack cannot deal with NAs in some context (e.g. next best dose) and therefore simulations runs into errors.

I think that rule=2 fix the problem (lowest or highest dose is returned when prob outside skel_probs), return what we want and is in line with the behavior of crmPack (to always return a dose).

I am not sure if there was a reason to choose rule=1 in the JSS paper, therefore I also ask @TJaki for input to this issue.

**Comment by wwojciech** [comment]
Hi @danielinteractive, @0liver0815 - after this merge I see a lot unit-test failures. Can we please undo this merge and solve it before merging into the `main`?

**Comment by danielinteractive** [comment]
@wwojciech No need to undo merge as we are not in a production environment. Can just make an issue with observations and then make a PR to fix

**Comment by wwojciech** [comment]
@danielinteractive sure, I think @0liver0815 did it already. Just a question now, because I created a child branch from this affected `main` for my issue and now many errors are thrown. Would you be having some suggestion on how to proceed? Tx!

**Comment by danielinteractive** [comment]
@wwojciech yes need to wait for the fix from @0liver0815 and then merge that to main and then update your feature branch by merging main into it

**Comment by wwojciech** [comment]
Perfect. Thanks a lot @danielinteractive!
